src/backend/columnar/sql/columnar--9.5-1--10.0-1.sql

```diff
@@ -3,34 +3,6 @@
 CREATE SCHEMA cstore;
 SET search_path TO cstore;
 
-CREATE FUNCTION cstore_fdw_handler()
-RETURNS fdw_handler
-AS 'MODULE_PATHNAME'
-LANGUAGE C STRICT;
-
-CREATE FUNCTION cstore_fdw_validator(text[], oid)
-RETURNS void
-AS 'MODULE_PATHNAME'
-LANGUAGE C STRICT;
-
-CREATE FOREIGN DATA WRAPPER cstore_fdw
-HANDLER cstore_fdw_handler
-VALIDATOR cstore_fdw_validator;
-
-CREATE FUNCTION cstore_ddl_event_end_trigger()
-RETURNS event_trigger
-AS 'MODULE_PATHNAME'
-LANGUAGE C STRICT;
-
-CREATE EVENT TRIGGER cstore_ddl_event_end
-ON ddl_command_end
-EXECUTE PROCEDURE cstore_ddl_event_end_trigger();
-
-CREATE FUNCTION pg_catalog.cstore_table_size(relation regclass)
-RETURNS bigint
-AS 'MODULE_PATHNAME'
-LANGUAGE C STRICT;
-
 CREATE TABLE cstore_data_files (
```

Comment:
wonder if we should add all objects to the pg12 and higher check now. Is there a benefit of precreating these object in pg11?

Or do we want to only not create objects that we can't create to keep the pg11->pg12 upgrade path minimal?